config/eks/config.go

```diff
@@ -52,6 +52,16 @@ func Configure(p *config.Provider) { // nolint:gocyclo
 			RefFieldName:      "SubnetIDRefs",
 			SelectorFieldName: "SubnetIDSelector",
 		}
+		r.References["launch_template.id"] = config.Reference{
+			Type:              "github.com/upbound/provider-aws/apis/ec2/v1beta1.LaunchTemplate",
+			RefFieldName:      "LaunchTemplateIDRefs",
+			SelectorFieldName: "LaunchTemplateIDSelector",
```

Comment:
```suggestion
			Type:              "github.com/upbound/provider-aws/apis/ec2/v1beta1.LaunchTemplate",
			TerraformName: "aws_kinesis_stream",
			Extractor:     common.PathTerraformIDExtractor,
```

The better way to use `TerraformName` even if it is difficult, in order not to be affected by any group changes. 
And we'd better not use `RefFieldName` and `SelectorFieldName` unless you have a specific reason to come up with it that way. It is used in the configuration you added as follows:
```
spec:
  launchTemplate:
    launchTemplateIdSelector:
      matchLabels:
        testing.upbound.io/example-name: ....
```
But as recommended:
```
spec:
  launchTemplate:
    IDSelector:
      matchLabels:
        testing.upbound.io/example-name: ....
```
What do you think?